commands/genpoke.js

```diff
@@ -6,25 +6,31 @@ const logger = require('../logs/logger.js');
 const CMD_TEMPLATE = '+genpoke [SPECIES] [LEVEL (1-20)] [NICKNAME - no spaces or special characters] [HIDDEN' +
 	' ABILITY % (as a number, 0-100)]';
 //help message
+const HELP_MESSAGE = '\n`+genpoke [species] [level (1-20)] [nickname] [Form Name] [hidden ability % (as a number, 0-100)]\n' +
+	'**FIELDS TO ADD LATER:**\n [trainerName] [private (optional: leave blank if public, put 1 if private]`' +
+	'\n\nCreates a new Pokemon when given the values above, printing it upon completion.\n\n';
+
+/*
 const HELP_MESSAGE = '\n' + CMD_TEMPLATE + '\n\n' + 'examples - `+genpoke Pikachu 1 Pika` or `+genpoke Pikachu 1' +
 	' Pika' +
 	' 30`' +
 	' (30% chance to have hidden ability)' +
 	'\n\nCreates a new Pokemon when given the values above, printing it upon completion. \n**Created as private by' +
 	' default** - use `+modpoke (name) private 0` to make publicly visible/editable\n\n' +
 	'(Hint: You can view an existing Pokemon with `+showpoke [nickname]`, or remove it using `+rempoke [nickname]`';
-
+*/
 module.exports.run = (client, connection, P, message, args) => {
 
 	let Pokemon = require('../models/pokemon.js');
 
 	if (args[0] === "help") {
+		message.reply('New Pokemon Generator. Variables in order:\n [Pokemon Species] [Level] [Pokemon Name] [Form Name] [Hidden Ability % (optional - CURRENTLY BROKEN)]').catch(console.error);
```

Comment:
Change to message.reply(CMD_TEMPLATE) - update CMD template as so:

'New Pokemon Generator. Command + variables in order: \n\n `+genpoke [POKEMON SPECIES] [LEVEL (1-20)] [NICKNAME] [FORM NAME] [HIDDEN ABILITY % (optional, 0-100)]`'